src/shared/component/header/Tabs.tsx

```diff
@@ -0,0 +1,147 @@
+// 로그인 여부에 따라 보여지는 탭 변경
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+import MyInfo from "./MyInfo";
+import Notification from "./Notification";
+
+interface TabsProps {
+  isLoggedIn: boolean;
+  headerColor: string;
+}
+
+const Tabs = ({ isLoggedIn, headerColor }: TabsProps) => {
+  const router = useRouter();
+
+  const [onToggle, setOnToggle] = useState("");
```

Comment:
[Lv2] onToggle 내부 데이터가 고정된 id로 되어있다면 해당 데이터들을 타입으로 관리하는게 좋습니다!

```suggestion

  type ToggleType = "notification" | "myInfo";
  const [onToggle, setOnToggle] = useState<ToggleType>();

  const onClickToggle = (target: ToggleType) => {
    setOnToggle((prev) => (prev === target ? undefined : target));
  };
  
```

Reply:
감사합니다! 확인했습니다 :)